src/CTA.Rules.PortCore/SolutionPort.cs

```diff
@@ -202,7 +202,7 @@ private void InitRules(List<PortCoreConfiguration> solutionConfiguration, List<A
                 if (projectConfiguration.UseDefaultRules)
                 {
                     //If a rules dir was provided, copy files from that dir into the rules folder
-                    if (!string.IsNullOrEmpty(projectConfiguration.RulesDir))
+                    if (!string.IsNullOrEmpty(projectConfiguration.RulesDir) && projectConfiguration.RulesDir != Constants.RulesDefaultPath)
```

Comment:
Why do we need this change? Please add an issue to describe what we're trying to address here.

Reply:
@marknfawaz Issue created [here](https://github.com/aws/cta/issues/184) with more details.